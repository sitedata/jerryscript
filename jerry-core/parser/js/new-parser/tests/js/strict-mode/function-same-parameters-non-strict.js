/* This test contains strict mode test for no
 * parameter name duplication is checked in non-strict mode.
 */

function strict_mode(param, param) {
}
