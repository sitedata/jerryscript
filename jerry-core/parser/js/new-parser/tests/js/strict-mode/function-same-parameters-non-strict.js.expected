
--- Script parsing start ---


--- Function parsing start ---

  Note: legacy (non-strict) argument definition


Final byte code dump:

  Maximum stack depth: 2
  Flags: [small_lit_enc,no_lexical_env]
  Argument range end: 2
  Register range end: 2
  Identifier range end: 2
  Const literal range end: 2
  Literal range end: 2

   0 : CBC_RETURN_WITH_BLOCK

Byte code size: 1 bytes

--- Function parsing end ---


Final byte code dump:

  Maximum stack depth: 0
  Flags: [small_lit_enc]
  Argument range end: 0
  Register range end: 0
  Identifier range end: 1
  Const literal range end: 1
  Literal range end: 2

   0 : CBC_INITIALIZE_VAR idx:0(ident)->var_ident(strict_mode) idx:1(lit)->function
   3 : CBC_RETURN_WITH_BLOCK

Byte code size: 4 bytes

Script parsing successfully completed. Total byte code size: 5 bytes

--- Script parsing end ---

